docs/crio.8.md

```diff
@@ -134,8 +134,19 @@ it later with **--config**. Global options will modify the output.
 **--default**
   Output the default configuration (without taking into account any configuration options).
 
+## FILES
+
+**crio.conf** (`/etc/crio/crio.conf`)
+  `cri-o` configuration file for all of the available command-line options for the crio(8) program, but in a TOML format that can be more easily modified and versioned.
+
+**registries.conf** (`/etc/containers/registries.conf`)
+  Registry configuration file specifies registries which are consulted when completing image names that do not include a registry or domain portion.
```

Comment:
We should add `registries.conf(5)` to “SEE ALSO” below, and see about (vendoring/)building/installing it in our CRI-O packages (source [here][1]).

[1]: https://github.com/containers/image/blob/fbc14df0f25a15b456c4f7ec69a1afbb19395544/docs/registries.conf.5.md

Reply:
wking no man page exists yet, at least that I can find.

Reply:
> wking no man page exists yet, at least that I can find.

I linked to [the source][1] above.  If `crio` consumers are getting their containers/image implementation from our vendored copy, I expect we should be vendoring that man-page source too and building/installing it along with our local docs.

[1]: https://github.com/containers/image/blob/fbc14df0f25a15b456c4f7ec69a1afbb19395544/docs/registries.conf.5.md

Reply:
> We don't currently install mounts.conf man page either. We are relying on other packages to provide that content.

That makes sense for me for our system dependencies.  But don't we [vendor containers/image][1] to isolate users from those system dependencies?  Maybe we need a packaging option that allows folks to use the system skopeo-containers package?  Is containers/image stable enough for us to float like that?

[1]: https://github.com/kubernetes-incubator/cri-o/tree/098559a713f19206f920322d6c44fc8bf7280e05/vendor/github.com/containers/image

Reply:
Hmm, actually I'm not sure we use this file at all.  It looks like 7a675ccd920 (#1150) may have [removed the only consumer from master][1].  It looks like we're configuring the image service via CRI-O-specific paths, e.g. `Registries` from #674 and `InsecureRegistries` from #577.  Should we drop this `registries.conf` reference?  I'm happy to file a PR to that effect, or leave it to folks who understand CRI-O configuration better and can confirm that we don't use this file.

[1]: https://github.com/kubernetes-incubator/cri-o/commit/7a675ccd9203173ed2eca117d5075a7a1e5b09c1#diff-7833748f87265413b28185455944f5a9L10

Reply:
@runcom Do you know if cri-o uses registries.conf?